src/audio/load.c

```diff
@@ -64,7 +64,7 @@ Acmd* alAdpcmPull(void* filter, s16* outp, s32 outCount, s32 sampleOffset, Acmd*
     inp = AL_DECODER_IN;
     aLoadADPCM(ptr++, f->bookSize, K0_TO_PHYS(f->table->waveInfo.adpcmWave.book->book));
 
-    looped = (outCount + f->sample > f->loop.end) && (f->loop.count != 0);
+    looped = (outCount + f->sample > (int)(f->loop.end)) && (f->loop.count != 0);
```

Comment:
For mismatched-sign comparisons like this, does the comparison get compiled to a signed or unsigned comparison? iirc comparing signed vs unsigned generates an unsigned comparison; if so, to suppress the warning in a way that is behavior-preserving you'd want to cast the signed arg to unsigned.

Though that raises the question, do we want to preserve original behavior here or do we actually want a signed comparison?

This consideration applies to some other sign-compare changes too